pandas/io/formats/style.py

```diff
@@ -1222,6 +1222,76 @@ class MyStyler(cls):
 
         return MyStyler
 
+    def pipe(self, func, *args, **kwargs):
+        """
+        Apply func(self, \*args, \*\*kwargs), and return the result.
+
+        .. versionadded:: 0.24.0
+
+        Parameters
+        ----------
+        func : function
+            function to apply to the Styler.
+            ``args``, and ``kwargs`` are passed into ``func``.
+            Alternatively a ``(callable, data_keyword)`` tuple where
+            ``data_keyword`` is a string indicating the keyword of
+            ``callable`` that expects the Styler.
+        args : iterable, optional
+            positional arguments passed into ``func``.
+        kwargs : mapping, optional
+            a dictionary of keyword arguments passed into ``func``.
+
+        Returns
+        -------
+        object : the value returned by ``func``.
```

Comment:
Is this not always returning a `pandas.Styler` object?

The description should go in the next line (indented), and without the colon. And it should start by a capital letter.

Reply:
The return type is determined by `func`, the user-provided function.